packages/valaxy/node/options.ts

```diff
@@ -70,7 +76,7 @@ export async function resolveOptions(options: ValaxyEntryOptions, mode: Resolved
 
   const { config: userValaxyConfig, configFile, theme } = await resolveValaxyConfig(options)
 
-  const themeRoot = getThemeRoot(theme, options.userRoot)
+  const themeRoot = getModuleRoot(`valaxy-theme-${theme}`, options.userRoot)
```

Comment:
`getMoudleRoot` can be internal.

`getThemeRoot` can do more about the theme in future. Maybe `@valaxyjs/theme-xxx`.